sp800-63b/sec5_authenticators.md

```diff
@@ -112,7 +112,7 @@ If the out of band authenticator sends an approval message over the secondary co
 
 Out of band verifiers SHALL generate a random authentication secret with at least 20 bits of entropy using an approved random number generator. They then optionally signal the device containing the subscriber's authenticator to indicate readiness to authenticate.
 
-Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP (or other software-based) service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
+Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP, chat, or other software-based service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
```

Comment:
The problem is that this introduces an impossible requirement: there is no way for the verifier to check if the phone number is forwarded to something other than the telephone network. But this is indeed another weakness in the use of SMS as proving possession and control of a specific authenticator.

At the same time, there are chat services, like perhaps Signal, that permit only one endpoint and detect and warn if that endpoint has changed. I can't say for sure that meets all of the requirements, but it's close enough that the specification should not rule out chat services entirely.